Makefile

```diff
@@ -13,8 +13,18 @@ GO_LDFLAGS=-ldflags "-X main.version=$(VERSION) \
 
 .PHONY: all test local-test
 
+AUTHENTICATED_ACCOUNT_ID=$(shell aws sts get-caller-identity --output text --query "Account")
+
+
 all: test
 
+local-run-controller: ## Run a controller image locally for SERVICE
+	@go run ./cmd/controller/main.go \
+		--aws-account-id=$(AUTHENTICATED_ACCOUNT_ID) \
+		--aws-region=us-west-2 \
+		--enable-development-logging \
+		--log-level=debug
```

Comment:
Consider putting these changes into a totally separate PR, as it's not germane to this PR.